helix-core/src/main/java/org/apache/helix/monitoring/metrics/AbnormalStatesMetricCollector.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.helix.monitoring.metrics;
+
+import javax.management.JMException;
+
+import org.apache.helix.HelixException;
+import org.apache.helix.monitoring.mbeans.MonitorDomainNames;
+import org.apache.helix.monitoring.metrics.implementation.RebalanceCounter;
+import org.apache.helix.monitoring.metrics.model.CountMetric;
+
+public class AbnormalStatesMetricCollector extends MetricCollector {
```

Comment:
Maybe some java doc for the public class to explain this monitor?

Reply:
The class name kind of explaining itself.

But it is a good idea to add some comment to the metric names.